correction-station-serial/configure.go

```diff
@@ -37,6 +37,8 @@ const (
 
 	svinModeEnable  = 0x01
 	svinModeDisable = 0x00
+
+	timeMode = "time"
```

Comment:
One follow up question - this was removed then added again, is this to set time mode and if so maybe a comment to explain its function a bit more.

Reply:
I moved this const from the station file to this one, but looking at it more closely I don't think this is actually being used, so removing. I didn't focus too much on the configure.go files but I can go back and clean them up in another PR.